### ISSUE TITLE
Transferring duplicates, better solution?

### ISSUE THREAD

**Issue by shiznid12** [PocketMobsters/PokeMobBot#230]
As of right now, we have a lot of settings to determine what to keep... we have global settings, and then we have specific settings for each pokemon individually.

My biggest gripe is... why is it 1000 CP OR 100%, why isn't it BOTH requirements need to be met? But with Pokemon like Chansey, she will never have that max CP.

So, why can't I tell the bot to only keep pokemon that have 95% IV and are above level 25? Right now, the cap for "wild" pokemon is level 30. Why do we care about CP, when CP is determined by level and IV primarily?

If my bot only kept Pokemon above level 25 and 95% IV, I could spend much less time "fiddling" with configs.

**Comment by Fenixxia** [comment]
I completely agree with this man. My offer was made a few days ago. But while "there are other priorities". Only what they need, if we have no core?

**Comment by Fenixxia** [comment]
Attack ID - https://docs.google.com/spreadsheets/d/1EyntqgeFI3fGPnnheucWuXyntOJvyZLRy6y4wIcvSk4/edit#gid=279241074

STAB for pokemon - https://docs.google.com/spreadsheets/d/1EyntqgeFI3fGPnnheucWuXyntOJvyZLRy6y4wIcvSk4/edit#gid=0

**Comment by Fenixxia** [comment]
This opinion too.  https://github.com/PocketMobsters/PokeMobBot/issues/4
and this https://github.com/PocketMobsters/PokeMobBot/issues/190
ofc and this https://github.com/PocketMobsters/PokeMobBot/issues/214

**Comment by eslindsey** [comment]
@Fenixxia this does not replace #231 because this changes something different.

**Comment by eslindsey** [comment]
Request rename: "Remove CP in favor of LV when comparing for transfer"

**Comment by littlemonster1** [comment]
sounds good to me, this would definitely be great.
however, there should be a bool function added then.
Some people might not want to filter for multiple options, e.g. at the lower levels you just want CP.

**Comment by xafelld** [comment]
"If my bot only kept Pokemon above level 25 and 95% IV, I could spend much less time "fiddling" with configs."

Yes please!!! Keep if Level AND %IV conditions are met :) 
Ideally being able to keep x # of each that match the criteria (so can evolve a few and get good move sets).
It would be so much easier to build a good team of pokers.

**Comment by chauhan-rahul** [comment]
I completely agree!!
